.github/workflows/reviewdog-workflow.yml

```diff
@@ -75,19 +75,19 @@ jobs:
           ref: ${{ github.event.pull_request.head.ref }}
           repository: ${{ github.event.pull_request.head.repo.full_name }}
           fetch-depth: 0
-        name: Install CPP Lint and ReviewDog
-      - run: |
-          wget -O - -q https://raw.githubusercontent.com/reviewdog/reviewdog/master/install.sh \
-          | sh -s -- -b .
-          pip install cpplint
-      - name: Run CPP Lint and push Annotations
-        env:
-          REVIEWDOG_GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: |
-          cpplint --recursive \
-            --filter=-build/include_subdir,-build/c++11,-build/include_what_you_use \
-            --linelength=120 ${{ github.workspace }} 2>&1 \
-            | ./reviewdog -efm="%f:%l: %m" -name="cpplint" -reporter="github-pr-review" -level="warning"
```

Comment:
You say the flags are kept equivalent but you removed `-level="warning"` and I believe the default is `-level="error"`. If this change was intentional, could you update the summary please?

Reply:
Good catch - updated the description.